src/Adapter/Order/CommandHandler/AddProductToOrderHandler.php

```diff
@@ -503,23 +603,25 @@ private function createNewInvoice(Order $order, Cart $cart, $isFreeShipping)
     /**
      * @param int $orderInvoiceId
      * @param Cart $cart
+     * @param Product $product
      *
      * @return OrderInvoice
      */
-    private function updateExistingInvoice($orderInvoiceId, Cart $cart)
+    private function updateExistingInvoice($orderInvoiceId, Cart $cart, array $products)
```

Comment:
```suggestion
    private function updateExistingInvoice($orderInvoiceId, Cart $cart, array $newProducts)
```

for better understanding

Reply:
Fixed